CMakeLists.txt

```diff
@@ -133,6 +133,7 @@ if(MPI_FOUND)
     logicalCartesianSize_and_refinement_test
     mapLevelIndicesToCartesianOutputOrder_test
     test_communication_utils
+    idEntity_idLevelZeroEntityRep_test
```

Comment:
The test name has been updated to id_entity_entityrep.cpp. I noticed that the surrounding tests all end with _test. I don’t mind either way, but it might be nice to keep the naming consistent. 🙂

Also, since these differences show up in CpGrid with LGRs, maybe we could move the test to tests/cpgrid/lgr/?